package.json

```diff
@@ -824,7 +824,8 @@
     "vscode:prepublish": "yarn compile",
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./",
-    "test": "jest",
+    "test": "jest unit/",
```

Comment:
`"test": "jest unit/",` is not ideal, I think [`jest` config](https://jestjs.io/docs/configuration) may help with that